easybuild/easyconfigs/o/OpenMM/OpenMM-8.1.2-foss-2023b-CUDA-12.4.0.eb

```diff
@@ -0,0 +1,67 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Update to 7.5.1
+# J. Sassmannshausen / GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '8.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://openmm.org'
+description = "OpenMM is a toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'opt': True}
+
+accept_eula = True
```

Comment:
This does not belong in the easyconfig file.
Accepting the eula is a "manual" step.